src/main/java/io/fabric8/jenkins/openshiftsync/CredentialsUtils.java

```diff
@@ -282,8 +284,33 @@ private static Credentials secretToCredentials(Secret secret) {
                     data.get(OPENSHIFT_SECRETS_DATA_USERNAME),
                     data.get(OPENSHIFT_SECRETS_DATA_SSHPRIVATEKEY));
         default:
-            logger.log(Level.WARNING,
-                    "Unknown secret type: " + secret.getType());
+			String type = secret.getType();
+			if (type == null || type.length() == 0) {
```

Comment:
why do we care about the type at all?  if it's not one of the special cases, can't we just always treat it as a textSecret, regardless of the "type"?

Reply:
i was worried about a non-null type that was incorrect, but the server side validation logic should catch that, right?

so yeah I'll remove the warning

Reply:
yeah, i'm not even sure if there a set of valid "types" or if it can be any value you want.

I think some types get special handling, but i suspect no type is "invalid"